frontend/app/utils/sin-utils.ts

```diff
@@ -29,6 +29,6 @@ export function isValidSin(sin: string): boolean {
  * @returns a formatted SIN using a supplied separator
  */
 export function formatSin(sin: string, separator = ' '): string {
-  if (!isValidSin(sin)) throw new Error('Invalid SIN format');
+  if (!isValidSin(sin)) return sin;
```

Comment:
It should throw an exception as it was before. If you only wan to execute `formatSin` if the SIN is defined and use Logical AND.

```
child.information?.socialInsuranceNumber && formatSin(child.information.socialInsuranceNumber)
```

Reply:
why though? SIN has already been validated by zod at this point if it's being rendered to the client.

Reply:
If it's the case then `formatSin` shouldn't throw an exception. If it does then there is a problem somewhere else.